src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -257,6 +257,11 @@ private static Option2<CodeStyleOption2<ExpressionBodyPreference>> CreatePreferE
             "csharp_style_prefer_primary_constructors",
             CSharpIdeCodeStyleOptions.Default.PreferPrimaryConstructors);
 
+        public static readonly Option2<CodeStyleOption2<bool>> PreferRemoveRedundantElseStatement = CreateOption(
+            CSharpCodeStyleOptionGroups.CodeBlockPreferences,
+            "csharp_style_prefer_remove_redundant_else_statement",
```

Comment:
this should probably be `dotnet_...` since it applies to VB as well.  csharp_style is only for constructs that are c# specific.

Reply:
Making that change only would fail this check since Option language is C# by default

https://github.com/dotnet/roslyn/blob/20f6ef9569054b497983b369e7c4ca651ac4b798/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/Option2.cs#L70

Should this option then go in some other file or just create it here with language as null?

Reply:
correct.  it goes into another file with teh rest of the dotnet style options.